lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1619,6 +1610,12 @@ public function get_item_schema() {
 				'type'        => 'boolean',
 				'context'     => array( 'view', 'edit' ),
 			);
+
+			$schema['properties']['password'] = array(
+				'description' => __( 'A password to protect access to the content and excerpt.' ),
+				'type'        => 'string',
+				'context'     => array( 'edit' ),
+			);
```

Comment:
I'm 👎  on `password` being a string or a boolean depending on context. As I mentioned above, I don't think `protected` at the top level makes as much sense because it's not saying the whole post is protected, however I don't mind too much on that.